zapcli/cli.py

```diff
@@ -415,6 +415,19 @@ def exclude_from_scanners(zap_helper, pattern):
         zap_helper.exclude_from_all(pattern)
 
 
+@cli.command('report', short_help='Generate XML or HTML report.')
+@click.option('--output', '-o', help='Output file for report.')
+@click.option('--output-format', '-f', default='xml', type=click.Choice(['xml', 'html']),
+              help='Report format.')
+@click.pass_obj
+def report(zap_helper, output, output_format):
+    """Generate XML or HTML report."""
+    if output_format == 'html':
+        zap_helper.html_report(output)
+    else:
+        zap_helper.xml_report(output)
```

Comment:
One thing I missed was that I think this should log an info message at the end to convey that the report was saved if the user doesn't have the `-v` flag on the command. But, I'll do that in a follow up commit. :)